guides/doc-Release_Notes/topics/katello-4.13.0.adoc

```diff
@@ -0,0 +1,136 @@
+* pass:[Hostgroup not showing associated Kickstart Repository in edit] - https://projects.theforeman.org/issues/37197[#37197]
+* pass:[Remove the setting 'upload_profiles_without_dynflow'] - https://projects.theforeman.org/issues/37195[#37195]
+* pass:[undefined method `family' for nil:NilClass after cloning a rhel8 host] - https://projects.theforeman.org/issues/37178[#37178]
+* pass:[Managing a Hosts Repository Sets does not behave as expected] - https://projects.theforeman.org/issues/37169[#37169]
+* pass:[Update Checkin time for ESXi hypervisors from virt-who report] - https://projects.theforeman.org/issues/37162[#37162]
+* pass:[Postgresql logs contain PG::UniqueViolation: ERROR: duplicate key value violates unique constraint "katello_available_module_streams_name_stream_context"] - https://projects.theforeman.org/issues/37137[#37137]
+* pass:[Offer a hint in the UI about how to get 'Synced Content' available] - https://projects.theforeman.org/issues/36992[#36992]
+* pass:[When cloning a hostgroup the fields content source content view and lifecycle are empty] - https://projects.theforeman.org/issues/35215[#35215]
+
+=== Inter Server Sync
+
+* pass:[content export actions are failing in ruby 3] - https://projects.theforeman.org/issues/37381[#37381]
+* pass:[cdn_ssl_version Setting enforces at most TLS1.0 version] - https://projects.theforeman.org/issues/36979[#36979]
+
+=== Notifications
+
+* pass:[Use with_enabled_email scope instead of handcrafting the query all over the place] - https://projects.theforeman.org/issues/37192[#37192]
+
+=== Reporting
+
+* pass:[Cannot create report "Host - All Installed Packages" for hosts running Debian/Ubuntu] - https://projects.theforeman.org/issues/37198[#37198]
+* pass:[SCA-Only: Remove Subscription-Entitlement notification] - https://projects.theforeman.org/issues/37170[#37170]
+
+=== Repositories
+
+* pass:[Repository synchronization progress does not get updated in real time on Satellite Web UI's "Content ---> Sync Status" page - RHEL8 Satellite 6.16] - https://projects.theforeman.org/issues/37442[#37442]
+* pass:[Upgrade pulp-container bindings to 2.20] - https://projects.theforeman.org/issues/37414[#37414]
+* pass:[Fix typo for container_repository_name in metadata_generate_needed?] - https://projects.theforeman.org/issues/37408[#37408]
+* pass:[Create a rake script that reindexes manifests with label information] - https://projects.theforeman.org/issues/37407[#37407]
+* pass:[Add Include Refs and Exclude Refs options for OSTree repository type] - https://projects.theforeman.org/issues/37383[#37383]
+* pass:[Container push can fail with a different JSON error] - https://projects.theforeman.org/issues/37380[#37380]
+* pass:[Index Pulp manifest annotations, labels, is_bootable, is_flatpak and expose them via API] - https://projects.theforeman.org/issues/37379[#37379]
```

Comment:
This is also useful information for users in release features. We not index metadata for container manifests which include anotations, labels and identifiers for flatpak and bootable images. The release includes a migration which will run on upgrades to fetch this metadata for existing container manifests in Katello.

Reply:
updated